usage/camp/src/main/java/io/brooklyn/camp/brooklyn/YamlLauncherAbstract.java

```diff
@@ -130,4 +126,13 @@ public void destroyAll() {
         }
     }
 
+   private Application createFromUrl(Reader reader) {
```

Comment:
The argument is no longer a URL, rename method. Also the TODO below is not needed in this case.